ui_advanced_config.py

```diff
@@ -25,116 +28,81 @@ def __init__(self, master=None, logger=None):
-        self.componentsTypeLabel = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeCombo = uiComboBox(master=self.componentsColumn)
-
-        self.componentsTypeAttr1Label = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeAttr1Entry = uiEntry(master=self.componentsColumn)
-        self.componentsTypeAttr2Label = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeAttr2Entry = uiEntry(master=self.componentsColumn)
-        self.componentsTypeAttr3Label = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeAttr3Entry = uiEntry(master=self.componentsColumn)
-        self.componentsTypeAttr4Label = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeAttr4Entry = uiEntry(master=self.componentsColumn)
-        self.componentsTypeAttr5Label = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeAttr5Entry = uiEntry(master=self.componentsColumn)
-        self.componentsTypeAttr6Label = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeAttr6Entry = uiEntry(master=self.componentsColumn)
-        self.componentsTypeAttr7Label = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeAttr7Entry = uiEntry(master=self.componentsColumn)
-
-        self.selectObjectsCombo = uiComboBox(master=self.objectsColumn)
-        self.objectsLabel = uiLabel(master=self.objectsColumn, text="Objects")
-        self.objectsUpdateButton = uiButton(
-            master=self.objectsColumn, command="", text="Update"
-        )
-        self.objectsIDLabel = uiLabel(master=self.objectsColumn, text="ID:")
-        self.objectsIDEntry = uiEntry(master=self.objectsColumn)
-        self.objectsNameLabel = uiLabel(master=self.objectsColumn, text="Name:")
-        self.objectsNameEntry = uiEntry(master=self.objectsColumn)
-        self.objectsFillAliveLabel = uiLabel(
-            master=self.objectsColumn, text="Fill Alive:"
-        )
-        self.objectsFillAliveEntry = uiEntry(
-            master=self.objectsColumn,
-        )
-        self.objectsFillDeadLabel = uiLabel(
-            master=self.objectsColumn, text="Fill Dead:"
+        self.agentFrame = uiQuietFrame(
+            master=self.teamsColumn, borderwidth=5, relief="ridge", sticky="nsew"
         )
-        self.objectsFillDeadEntry = uiEntry(master=self.objectsColumn)
-        self.objectsTextLabel = uiLabel(master=self.objectsColumn, text="Text:")
-        self.objectsTextEntry = uiEntry(master=self.objectsColumn)
-        self.objectsHealthLabel = uiLabel(master=self.objectsColumn, text="Health:")
-        self.objectsHealthEntry = uiEntry(master=self.objectsColumn)
-        self.objectsDensityLabel = uiLabel(master=self.objectsColumn, text="Density:")
-        self.objectsDensityEntry = uiEntry(master=self.objectsColumn)
-        self.objectsCompIDsLabel = uiLabel(master=self.objectsColumn, text="Comp IDs:")
```

Comment:
I'm not sure I understand how to actually add a component to an object? I'm not sure if this is a critic of function or UI/UX, because I'm not entirely sure what is expected to be happening.